### DISCUSSION TITLE
evga b-stock list

### DISCUSSION THREAD

**Discussion by cmil6199** [jef/streetmerchant#2448] [Ideas and feature requests]
I am trying to modify the evga store portion to search the b-stock page. I want to know when they post new items in a particular group.

https://www.evga.com/products/ProductList.aspx?type=8&family=GeForce+30+Series+Family
https://www.evga.com/products/ProductList.aspx?type=8&family=GeForce+20+Series+Family

They post b-stock usually on wednesday and friday, but I always seem to miss them.

Any help would be great, I am sure others would use this too.

Thanks,
Chris

**Comment by cmil6199** [comment]
I just need it to trigger the popup when anything on that page has the add to cart button.

**Reply by helmsley99** [reply]
I keep getting blocked by them? Any advice?

**Comment by Mr-Tech-13** [comment]
Does it matter to you if it is just 30 series or 20 series? If it's just 30 series, I can send you a link to my bot
